homeassistant/components/transmission/sensor.py

```diff
@@ -183,6 +183,7 @@ def _torrents_info(torrents, order, statuses=None):
             "percent_done": f"{torrent.percentDone * 100:.2f}",
             "status": torrent.status,
             "id": torrent.id,
+            "ratio": torrent.uploadRatio,
```

Comment:
Was this a bug?

Reply:
Good catch - I will remove from this PR since it belongs in the next refactor!